presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -56,6 +56,16 @@ String Functions
 
     Removes trailing spaces from ``string``.
 
+.. function:: split(string, delimiter) -> array<varchar>
+
+    Splits ``string`` on ``delimiter`` and return an array.
+
+.. function:: split(string, delimiter, limit) -> array<varchar>
+
+    Splits ``string`` on ``delimiter`` and return an array of size at most
+    ``limit``. The last element in the array always contain everything
```

Comment:
Actually, that sounds like reasonable behavior. It's what most other languages do (ruby, java, etc).